vite.config.ts

```diff
@@ -11,4 +11,14 @@ export default defineConfig({
     host: "localhost",
     port: 3000,
   },
+  build: {
+    rollupOptions: {
+      onwarn(warning, warn) {
+        if (warning.code === "MODULE_LEVEL_DIRECTIVE") {
+          return;
+        }
+        warn(warning);
+      },
+    },
+  },
```

Comment:
Don't believe this is necessary anymore.  Don't want to leave in things that ignore possible future warnings...